collector/nginx_plus.go

```diff
@@ -289,6 +302,24 @@ func (c *NginxPlusCollector) Collect(ch chan<- prometheus.Metric) {
 		ch <- prometheus.MustNewConstMetric(c.streamUpstreamMetrics["zombies"],
 			prometheus.GaugeValue, float64(upstream.Zombies), name)
 	}
+
+	for name, zone := range stats.StreamZoneSync.Zones {
+		ch <- prometheus.MustNewConstMetric(c.streamZoneSyncMetrics["records_pending"],
+			prometheus.CounterValue, float64(zone.RecordsPending), name)
+		ch <- prometheus.MustNewConstMetric(c.streamZoneSyncMetrics["records_total"],
```

Comment:
this is a gauge

Reply:
You mean this can go down too? When, when an NGINX Plus instance is stopped and run again?